server/plugin.go

```
@@ -82,10 +82,17 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 
 		if origText != "" {
 			newText := origText
+
+			channel, _ := p.API.GetChannel(post.ChannelId)
```

3/5 that you should not ignore the error here.